concent_api/core/tests/test_unit_validation.py

```diff
@@ -1,9 +1,11 @@
 from unittest import TestCase
-
+from assertpy import assert_that
 import mock
+import pytest
+
 from django.test import override_settings
 
-from golem_messages.factories.tasks import TaskToComputeFactory
+from golem_messages.factories.tasks import TaskToComputeFactory, ComputeTaskDefFactory
```

Comment:
Imports should be split into separate lines.

Reply:
done